### PR TITLE
error messages for instructor show

### PR DESCRIPTION
**Pull request by coati-00**
havent figured out how to remove them yet though

### CODE REVIEW

File: media/js/ccnmtljs/bb_control_views.js [hunk 1]
```diff
@@ -183,78 +190,122 @@ var ManageInstructorsView = Backbone.View.extend({
...
+         
+        if(this.validAddForm())
+        {
+            this.instructor_collection_view.collection.create(
+            {
+                first_name : jQuery(".instructor-frst-name").val(),
+                last_name : jQuery(".instructor-last-name").val(),
+                email : jQuery(".instructor-email").val()
+            },
+            {
```
Comment (sdreher) [inline media/js/ccnmtljs/bb_control_views.js]: Are these success and error functions misplaced?

File: media/js/ccnmtljs/bb_control_views.js [hunk 2]
```diff
@@ -183,78 +190,122 @@ var ManageInstructorsView = Backbone.View.extend({
...
+        if(this.validAddForm())
+        {
+    	    this.student_collection_view.collection.create(
     	    {   
     	        	first_name : jQuery(".frst-name").val(),
     	            last_name : jQuery(".last-name").val(),
     	            email : jQuery(".email").val()
     	    },
     	    {
+    	    	success: function(model, response){
```
Comment (sdreher) [inline media/js/ccnmtljs/bb_control_views.js]: These success & errors also seem to be declared inline..

File: media/js/ccnmtljs/bb_control_views.js [hunk 3]
```diff
@@ -329,17 +373,17 @@ var TeamControlView = Backbone.View.extend({
 	    {
     	    success: function(model, response)
```
Comment (sdreher) [inline media/js/ccnmtljs/bb_control_views.js]: Ditto here with the inline success/error functions

Comment (sdreher) [inline media/js/ccnmtljs/bb_control_views.js]: Ditto here with the inline success/error functions

File: media/js/ccnmtljs/bb_control_views.js [hunk 4]
```diff
@@ -172,9 +178,10 @@ var ManageInstructorsView = Backbone.View.extend({
     	}
     	if((jQuery(".add-instructor-frm input.instructor-email").val().length) === 0)
     	{
+    		is_valid = false;
     		if((jQuery(".inst-email").has('b').length) === 0)
     		{
-    			jQuery(".inst-email").append("<b>Please enter a email.</b>").css('color', 'red');
+    			jQuery(".inst-email").append("<b class='error-msg'>Please enter a email.</b>").css('color', 'red');
```
Comment (sdreher) [inline media/js/ccnmtljs/bb_control_views.js]: "an" email

File: media/js/ccnmtljs/bb_control_views.js (hunk 1 above)
Comment (coati-00) [inline media/js/ccnmtljs/bb_control_views.js]: no? The request occurs when a collection object is created?

**Pull request merged**